lib/core/repository/cms/attribute_builder.rb

```diff
@@ -6,6 +6,22 @@ def self.build(response)
 
         attributes['title']       = attributes['label']
         attributes['body']        = BlockComposer.new(attributes['blocks']).to_html
+
+        attributes['blocks'].select {|h| h['identifier'].start_with?('raw_') }.each do |h|
+          key = h['identifier'].gsub(/^raw_/, '')
+          attributes[key] = h['content']
+        end
+
+        attributes.select {|k,v| k.match(/_(\d+)_/) }.each do |k,v|
+          _, object, number, field = k.match(/(\w+)_(\d+)_(\w+)/).to_a
+          number = number.to_i - 1
+          key = object.pluralize
+
+          attributes[key] ||= []
+          attributes[key][number] ||= {}
+          attributes[key][number][field] = v
+        end
+
```

Comment:
I know that it is just assignments, but can you move this to private methods, please?